solr/core/src/java/org/apache/solr/handler/component/RealTimeGetComponent.java

```diff
@@ -1164,6 +1225,31 @@ public void processGetUpdates(ResponseBuilder rb) throws IOException
     return new ArrayList<>(versionsToRet);
   }
 
+  /**
+   *  <p>
+   *    Lookup strategy for {@link #getInputDocument(SolrCore, BytesRef, AtomicLong, Set, Resolution)}.
+   *  </p>
+   *  <ul>
+   *    <li>{@link #FULL_DOC}</li>
+   *    <li>{@link #DOC_CHILDREN}</li>
+   *    <li>{@link #FULL_HIERARCHY}</li>
+   *  </ul>
+   */
+  public static enum Resolution {
```

Comment:
Suggested new names:  "DOC", "DOC_WITH_CHILDREN", "ROOT_WITH_CHILDREN".
Resolution is now only used where there is no "in place" so no need to mention partial documents.